### PR TITLE
[v11] Add support for Cassandra audit events

### PR DESCRIPTION
**Pull request by smallinsky**
Backport https://github.com/gravitational/webapps/pull/1240 to `teleport-v11`

### Render:
Screenshot 2022-10-10 at 19 03 27

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

File: packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap [hunk 1]
```diff
@@ -1009,7 +1157,7 @@ exports[`list of all events 1`] = `
         <td
           style="word-break: break-word;"
         >
-          User [alice] has send RPC Request [undefined] in database [master] on [sqlserver02]
+          User [alice] has sent RPC Request [undefined] in database [master] on [sqlserver02]
```
Comment (kimlisa) [inline packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap:1160]: could you check why this has an `undefined` value? its either our JSON fixture is incorrect, or we are accessing an incorrect field name

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 2]
```diff
@@ -906,7 +906,34 @@ export const formatters: Formatters = {
...
+      `User [${user}] has sent Cassandra Prepare [${truncateStr(
+        query,
+        80
+      )} to [${db_service}]`,
+  },
+  [eventCodes.CASSANDRA_EXECUTE_EVENT]: {
+    type: 'db.session.cassandra.execute',
+    desc: 'Cassandra Execute',
+    format: ({ user, db_service }) =>
+      `User [${user}] has sent Cassandra Execute to ${db_service}`,
```
Comment (kimlisa) [inline packages/teleport/src/services/audit/makeEvent.ts:930]:
```suggestion
      `User [${user}] has sent Cassandra Execute to [${db_service}]`,
```

**Review by kimlisa — COMMENTED** [review]
probably need to forward port after changes. i suggest next time, with any UI changes, i would always include screenshots of the additions (easier to catch stuff)

**Comment by smallinsky** [comment]
@kimlisa Thanks for suggestion. 
I didn't know about `yarn storybook`  earlier.

File: packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap (hunk 1 above)
Comment (smallinsky) [reply packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap]: Fixed the templated was outdated after switching from `porcname` to `proc_name`

**Comment by kimlisa** [comment]
@smallinsky also what browser and OS are you using? the screenshot says everything is out of alignment (nothing to do with your changes)

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 3]
```diff
@@ -906,7 +906,34 @@ export const formatters: Formatters = {
...
+      `User [${user}] has sent Cassandra Batch to [${db_service}]`,
+  },
+  [eventCodes.CASSANDRA_PREPARE_EVENT]: {
+    type: 'db.session.cassandra.prepare',
+    desc: 'Cassandra Prepare Event',
+    format: ({ user, db_service, query }) =>
+      `User [${user}] has sent Cassandra Prepare [${truncateStr(
+        query,
+        80
+      )} to [${db_service}]`,
```
[the remark this answers is not in the document]
Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts:924]: i don't think this is resolved? the missing closing bracket?

**Review by kimlisa — APPROVED** [review]
after one minor comment

File: packages/teleport/src/services/audit/makeEvent.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (smallinsky) [reply packages/teleport/src/services/audit/makeEvent.ts]: Indeed, I have missed that. Thanks.

**Pull request merged**